src/tools/compiletest/src/runtest.rs

```diff
@@ -2359,11 +2359,6 @@ impl<'test> TestCx<'test> {
     }
 
     fn run_rmake_test(&self) {
-        // FIXME(#11094): we should fix these tests
-        if self.config.host != self.config.target {
```

Comment:
hm, won't this make us run more tests now? Or do we handle this elsewhere for fulldeps suites somehow?

Reply:
I think this was added long long ago when we still ran more tests in more situations (like tests on dist builders), but nowadays I don't think this is ever triggered. If I'm wrong though this'll just fail in CI